main.go

```diff
@@ -41,8 +41,12 @@ type Options struct {
 		RampTime   time.Duration `long:"ramptime" description:"duration to spend ramping up or down to the desired TPS" default:"1s"`
 	} `group:"Quantity Options"`
 	Output struct {
-		Sender   string `long:"sender" description:"type of sender" choice:"honeycomb" choice:"otel" choice:"print" choice:"dummy" default:"honeycomb"`
-		Protocol string `long:"protocol" description:"for otel only, protocol to use" choice:"grpc" choice:"protobuf" choice:"json" default:"grpc"`
+		Sender             string        `long:"sender" description:"type of sender" choice:"honeycomb" choice:"otel" choice:"print" choice:"dummy" default:"honeycomb"`
+		Protocol           string        `long:"protocol" description:"for otel only, protocol to use" choice:"grpc" choice:"http" default:"grpc"`
+		MaxQueueSize       int           `long:"maxqueuesize" description:"for otel only, maximum number of spans to queue before dropping" default:"0"`
+		MaxExportBatchSize int           `long:"maxexportbatchsize" description:"for otel only, maximum number of spans to export at once" default:"0"`
+		BatchTimeout       time.Duration `long:"batchtimeout" description:"for otel only, maximum time to wait before sending a batch" default:"0s"`
+		ExportTimeout      time.Duration `long:"exporttimeout" description:"for otel only, maximum time to wait for a batch to be sent" default:"0s"`
```

Comment:
```suggestion
		ExportTimeout      time.Duration `long:"exporttimeout" description:"for otel only, maximum time to wait for a batch send to be completed" default:"0s"`
```

Also same comment as above -- 0 means something, so maybe for documentation we set the default to be an actual value, so you know when it happens?